spec/ims/lti/models/image_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+
+module IMS::LTI::Models
+  describe Image do
+
```

Comment:
Image is just a PORO with some inherited methods that are tested in the superclass